### ISSUE TITLE
How to force SSL on Heroku

### ISSUE THREAD

**Issue by paulspencerwilliams** [luminus-framework/luminus#282]
I've pushed a playground app to Heroku and have enabled SSL using 

````heroku ps:type hobby````

I'd now like to force SSL which [needs to be done at the application level](https://help.heroku.com/J2R1S4T8/can-heroku-force-an-application-to-use-ssl-tls). How can this be done in Luminus?

**Comment by yogthos** [comment]
Yeah, you could add a piece of middleware that redirects to SSL. [This](https://github.com/ring-clojure/ring-ssl) middleware should do the trick.
